ui/frontend/actions.ts

```diff
@@ -2,7 +2,13 @@ import fetch from 'isomorphic-fetch';
 import { ThunkAction as ReduxThunkAction } from 'redux-thunk';
 import url from 'url';
 
-import { clippyRequestSelector, getCrateType, isAutoBuildSelector, runAsTest } from './selectors';
+import {
+  clippyRequestSelector,
+  formatRequestSelector,
```

Comment:
Thanks for following this new pattern; I know it can be easy to just leave the code as it was! ❤️